src/Likvido.Test.Api/HttpClientFactory/HttpClientFactoryOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Likvido.Test.Api
+{
+    public class HttpClientFactoryOptions
+    {
+        public Action<IServiceCollection>? ConfigureServices { get; set; }
+#pragma warning disable CA2227 // Collection properties should be read only
+        public Dictionary<string, Action<HttpClient, IConfiguration>>? ConfigureNamedHttpClients { get; set; }
```

Comment:
1. `IServiceProvider` gives much better flexibility
2. I'm suggesting to change parameters order to match `AddHttpClient` extension method
```suggestion
        public Dictionary<string, Action<IServiceProvider, HttpClient>>? ConfigureNamedHttpClients { get; set; }
```

Reply:
👍 updated